developer_manual/design/navigation.rst

```diff
@@ -43,6 +43,14 @@ Basic layout
         </ul>
     </div>
 
+
+Basic rules
+=============
+
+* You can **not** change the default padding of the navigation elements.
+* We encourage you to add icons on every top-level item of your navigation for accessibility.
+* Do **not** override the default structure and/or CSS. Everything has been carefully tuned.
```

Comment:
The first point is kind of a very specific duplicate of this. Should we instead use this sentence, and expand the end:
»Everything has been carefully tuned for usability and accessibility.«?

And the icons are more for »recognizability« than »accessibility«

Reply:
Recognizability seems like a very complex and translated word! ^^
What about the word 'identification'?